src/utils/chartOptions.js

```diff
@@ -171,47 +164,132 @@ export const barGraphOptions = options => merge({
  * Options ONLY for Doughnut chart
  * @param {object} options - More options that can be pass to the chart
  */
-export const doughnutGraphOptions = options => merge({
+export const doughnutChartOptions = options => merge({
   cutoutPercentage: 60,
 
   elements: {
     arc: {
-      backgroundColor: styles.ultramarineBlue,
+      backgroundColor: chartStyles.ultramarineBlue,
       borderAlign: 'center',
-      borderColor: styles.whiteColor,
+      borderColor: chartStyles.whiteColor,
       borderWidth: 1,
     },
   },
 }, baseOptions, options);
 
 
+/**
+ * data ONLY for Line chart
+ * @param {object} data - More data that can be pass to the chart
+ */
+export const lineChartData = data => merge({
+  datasets: [
+    {
+      backgroundColor: chartStyles.transparent,
+      borderColor: chartStyles.borderColor,
+      pointBorderColor: chartStyles.borderColor,
+      pointBackgroundColor: chartStyles.whiteColor,
+      pointHoverBackgroundColor: chartStyles.whiteColor,
+      pointHoverBorderColor: chartStyles.ultramarineBlue,
+      pointHoverBorderWidth: 4,
+      borderWidth: 2,
+    },
+  ],
+}, data);
+
+
+/**
+ * data ONLY for Bar chart
+ * @param {object} data - More data that can be pass to the chart
+ */
+export const barChartData = data => merge({
+  datasets: [
+    {
+      barPercentage: 1,
+      categoryPercentage: 0.5,
+      barThickness: 'flex',
+      maxBarThickness: 14,
+      minBarLength: 0,
+    },
+    {
+      barPercentage: 1,
+      categoryPercentage: 0.5,
+      barThickness: 'flex',
+      maxBarThickness: 14,
+      minBarLength: 0,
+      stack: 'dob',
+      backgroundColor: chartStyles.ufoGreen,
+    },
+  ],
+}, data);
```

Comment:
There’s just one detail to fix: it should handle different lengths of dataset, because now you cannot get rid of the undefined.
<img width="656" alt="Screenshot 2019-11-18 at 15 01 14" src="https://user-images.githubusercontent.com/1254342/69059283-80388480-0a15-11ea-8cb1-d820cc0fa909.png">


You can do something like this:
export const barChartData = data => merge({
  datasets: (data.datasets || []).map((_, i) => ({
      barPercentage: 1,
      categoryPercentage: 0.5,
      barThickness: 'flex',
      maxBarThickness: 14,
      minBarLength: 0,
      backgroundColor: primaryColorPalete[i],
    })),
}, data);